composer.json

```diff
@@ -2,12 +2,12 @@
   "name": "tawk/widget",
   "description": "Tawk live chat widget",
   "type": "magento2-module",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
The module definition lists 1.3.1 as version of module
Version here should be brought in line with version listed in module and the overall bumped to 1.4.0